package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "platformsh-client",
-  "version": "0.1.185",
+  "version": "0.1.186",
```

Comment:
@Ndifreke could you roll back this version bump? it will be done when the new version will be released.